modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,379 @@
+
+For example, if you produce to a topic according to the following Avro schema:
+
+```
+{
+    "type": "record",
+    "name": "ClickEvent",
+    "fields": [
+        {
+            "name": "user_id",
+            "type": "int"
+        },
+        {
+            "name": "event_type",
+            "type": "string"
+        },
+        {
+            "name": "ts",
+            "type": "string"
+        }
+    ]
+}
+
+```
+
+The `key_value` mode writes to the following table format:
+
+```
+CREATE TABLE ClickEvent (
+  redpanda OBJECT(
+    partition INT,
+    timestamp TIMESTAMP,
+    offset    LONG,
+    headers   ARRAY(OBJECT(KEY BINARY, VALUE BINARY NULLABLE)),
+    key       BINARY NULLABLE,
+    data      BINARY NULLABLE
+  )
+)
+```
+
+Consider this schemaless approach if the topic data is in JSON, or if you can use the Iceberg data in its semi-structured format.
+
+The `value_schema_id_prefix` mode translates to the following table format:
+
+```
+CREATE TABLE ClickEvent (
+  redpanda OBJECT(
+    partition INT,
+    timestamp TIMESTAMP,
+    offset    LONG,
+    headers   ARRAY(OBJECT(KEY BINARY, VALUE BINARY NULLABLE)),
+    key       BINARY NULLABLE
+  ),
+   user_id INT,
+   event_type STRING NULLABLE,
+   ts STRING NULLABLE
+);
+
+```
+
+With schema integration, Redpanda uses the schema ID prefix embedded in each record to find the matching schema in the Schema Registry. Producers to the topic must use the schema ID prefix in the serialization process so Redpanda can determine the schema used for each record, parse the record according to that schema, and use the schema for the Iceberg table as well.
+
+=== Schema types translation
+
+[tabs]
+======
+Avro::
++
+--
+* Redpanda supports direct translations of the following Avro types to Iceberg value domains:
+** boolean
+** int
+** long
+** float
+** double
+** bytes 
+** string
+** record
+** array
+** maps
+** fixed
+** decimal
+** uuid
+** date
+** time
+** timestamp
+* Different flavors of timestamp and time types are all represented by the same Iceberg type, conversion will be done.
+* Avro unions are flattened to structs with optional fields, out of which only one is present, or none in case there is a field with null type:
+** For example, the union `["int", "long", "float"]` is represented as an Iceberg struct `OBJECT(0 INT NULLABLE, 1 LONG  NULLABLE, 2 FLOAT NULLABLE)`.
+** The union `["int", null, "float"]` is represented as an Iceberg struct `OBJECT(0 INT NULLABLE, 1 FLOAT NULLABLE)`.
+* All fields are required by default (Avro always sets a default in binary representation).
+* Duration logical type is ignored.
+* Null type is ignored and not represented in Iceberg schema.
+* Recursive types are not supported.
```

Comment:
It's probably going to be easier for people to grok this as a table of

| Avro Type | Iceberg Type |
|--------|--------|
| Long | Long |
| Float | Float |
| Bytes | Bytes |

With notes at the bottom talking about unions, timestamps, time, nulls, recursive types, etc.

Something like this for inspiration: https://avro.apache.org/docs/1.11.1/specification/#schema-record